README.rst

```diff
@@ -308,7 +312,7 @@ There is also an analytics object with each ``HyperparameterTuner`` instance, wh
 
 For more detailed examples of running hyperparameter tuning jobs, see: https://github.com/awslabs/amazon-sagemaker-examples.
 
-For more detailed explanations of the classes mentioned, see:
+For more detailed explanations of the HyperparameterTuner and analytics classes, see:
```

Comment:
one thing that I probably didn't call out explicitly enough is that we have classes for parameter ranges, which are also included in the first link. not sure if it's worth also mentioning here since it's in addition to the `HyperparameterTuner` and analytics classes.

also, might want to put backticks around 'HyperparameterTuner'